core/processor/src/main/java/io/quarkus/annotation/processor/generate_doc/ConfigPhase.java

```diff
@@ -37,6 +39,20 @@ public int compare(ConfigPhase firstPhase, ConfigPhase secondPhase) {
                             return -1;
                     }
                 }
+                case BOOTSTRAP: {
```

Comment:
Heh! This bit of code probably should not have been approved - instead the phases probably ought to have been reordered and the default comparison used.... but I guess that can be a fix for another day.